packages/ketcher-polymer-editor-react/src/components/shared/StructRender/index.tsx

```diff
@@ -0,0 +1,41 @@
+/****************************************************************************
```

Comment:
Can't we use this component directly from `ketcher-core`?

Reply:
This is simply a container for ketcher-core render